workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -17,7 +17,7 @@ public class Bootstrap : MonoBehaviour
 
         private const int TargetFrameRate = -1; // Turns off VSync
 
-        public const string LoggerName = "Bootstrap";
+        public const string LoggerName = nameof(Bootstrap);
```

Comment:
we should probably define whether we prefer strings or nameof. In another PR of mine, some people preferred to use strings directly, though I would say that nameof is less error-prone

Reply:
Yeah, a string constant isn't refactor-proof, whereas `nameof` is.